### ISSUE TITLE
Accessibility (a11y) template option for ARIA-LIVE

### ISSUE THREAD

**Issue by vcarreno96** [angular-ui/ui-router#2340]
Wondering if it would be possible to allow the ui-view directive to not replace its entire contents with a template. Reason would be to assist screen readers to detect the ARIA-LIVE attribute to speak dynamic content changes to the user. EX:

``` html
<div ui-view>
      <div aria-live="polite"></div>
</div>
```

If the descendant div is preserved, the screen reader will be able to detect html changes to its ancestor nodes. Unfortunately, aria-live does not work if changes occur on descendant nodes so adding the attribute to the ui-view directive does not work. Testing using JAWS v.15 screen reader and IE 11.

**Comment by nateabele** [comment]
That's interesting. I don't think there's currently a way to do this, but it's something I'd be interested in exploring. I don't think there's a really great way to extend directives in the way that you're talking about.

**Comment by vcarreno96** [comment]
Yes, it's challenging. I'll see if I can fabricate an example of replacing the content at an ancestor level DOM element to verify that the ARIA-LIVE attribute works as expected. Don't want to put you guys through trouble if the reader still does not detect the dynamic content changes.
